scripts/o3de/o3de/engine_template.py

```diff
@@ -2816,6 +2895,45 @@ def add_args(subparsers) -> None:
     create_gem_subparser.set_defaults(func=_run_create_gem)
 
 
+    create_repo_subparser = subparsers.add_parser('create-repo')
+    create_repo_subparser.add_argument('-rp','--repo-path', type=pathlib.Path, required=True,
+                                       help = 'The location of the remote repo you wish to create from the RemoteRepo template,'
+                                       'can be an absolute path or relative to the current working directory'
+                                       'Ex. C:/o3de/TestRemoteRepo'
+                                       'TestRemoteRepo = <RemoteRepo_name> if --project_name not provided')
+    create_repo_subparser.add_argument('-ru', '--repo-uri', type=str, required=True,
+                                       help = 'The online remote repository uri')
+    create_repo_subparser.add_argument('-rn', '--repo-name', type=str, required=False,
+                                       help = 'The name to use when substituting the ${Name} placeholder for the remote repo,'
+                                           ' must be alphanumeric, '
+                                           ' and can contain _ and - characters.'
+                                           ' If no name is provided, will use last component of the repo-path provided by user.'
+                                           ' Ex. Remote_repo1')
```

Comment:
Actually, I think the repo name can contain spaces and non-alphanumeric characters.
```suggestion
                                       help = 'The name to use when substituting the ${Name} placeholder for the remote repo,'
                                           ' If no name is provided, will use last component of the repo-path provided by user.'
                                           ' Ex. RemoteRepo1')
```